scio-bigquery/src/main/scala/com/spotify/scio/bigquery/BigQueryScioContext.scala

```diff
@@ -54,10 +54,39 @@ final class BigQueryScioContext(@transient val self: ScioContext) extends Serial
     self.read(BigQueryTable(tableSpec))
 
   /**
-   * Get a typed SCollection for a BigQuery SELECT query or table.
+   * Get an SCollection for a BigQuery table using the storage API.
+   *
+   * @param selectedFields names of the fields in the table that should be read. If empty, all
+   *                       fields will be read. If the specified field is a nested field, all the
+   *                       sub-fields in the field will be selected.
+   * @param rowRestriction SQL text filtering statement, similar ti a WHERE clause in a query.
+   *                       Currently, we support combinations of predicates that are a comparison
+   *                       between a column and a constant value in SQL statement. Aggregates are
+   *                       not supported. For example:
+   *
+   * {{{
+   * "a > DATE '2014-09-27' AND (b > 5 AND c LIKE 'date')"
+   * }}}
+   */
+  def bigQueryStorage(tableSpec: String,
+                      selectedFields: List[String] = Nil,
+                      rowRestriction: String = null): SCollection[TableRow] =
```

Comment:
We can still keep the `null` here for the sake of "usability" and because it's consistent with the rest of the api ... but I think we could at least represent it as `Option` internally. wdyt?